stories/Chip.stories.js

```diff
@@ -12,6 +13,72 @@ stories.addParameters({
   }
 });
 
-stories.add('Default', () => <Chip>Jane Doe</Chip>);
+stories.addDecorator(withKnobs);
 
-stories.add('with close', () => <Chip close>Jane Doe</Chip>);
+stories.add('Default', () => <Chip>{text('text', 'Jane Doe')}</Chip>);
+
+stories.add('Tags', () => <Chip close={boolean('close', true)}>Tag</Chip>);
+
+stories.add('Contacts', () => (
+  <Chip>
+    <img
+      src={text('image', 'https://materializecss.com/images/yuna.jpg')}
+      className="responsive-img"
+      alt="Contact Person"
+    />
+    {text('text', 'Jane Doe')}
+  </Chip>
+));
+
+stories.add('Javascript Plugin', () => (
```

Comment:
To add notes use this approach https://github.com/react-materialize/react-materialize/blob/master/stories/Autocomplete.stories.js#L24